README.md

```diff
@@ -1,13 +1,21 @@
 # Project Happy Thoughts API
+This week's project was to use our new skills with Express and Mongodb to build an API which includes both GET request endpoints to return data and POST endpoints to create data. This API is then used for my ❤️ Happy Thought ❤️ project where users can post their thoughts. 
 
-Replace this readme with your own information about your project.
+## What I have learned 🧠
+- How to build a registration flow
+- How to handle authentication, both in the frontend and in the backend
+- How to build a frontend and backend at the same time
 
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+## Endpoints
+- GET '/': returns the list of all endpoints;
+- GET '/thoughts': returns a list of all;
+- POST '/thoughts': posts for message, username and hashtag;
+- POST '/'/thoughts/:thoughtId/like': posts for likes; 
+
+## View it live
+- BE: https://poggi-happy-thoughts-api.herokuapp.com/
+- FE: https://poggi-happy-thoughts.netlify.app/
```

Comment:
Great documentation! Very clear description of en points. Only the last one I think has too many slashes.